external/contracts/set/ConstantPriceAdapter.sol

```diff
@@ -0,0 +1,74 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity 0.8.17;
+
+/**
+ * @title ConstantPriceAdapter
+ * @author Index Coop
+ * @notice Price adapter contract for AuctionRebalanceModuleV1 that returns a constant price.
+ * The rate of change is zero.
+ * Price formula: price = initialPrice
+ */
+contract ConstantPriceAdapter {
```

Comment:
Might be a good idea to breakout the interface that the module expects and have this inherit/implement that interface. Mostly for completeness so somewhat optional, but if the idea is that we make more adapters... would be good to start on the right foot.

Reply:
leaving for now since i only added for unit testing, will hopefully setup better in a new repo 👀